js/components/portal-dashboard/feedback/rubric-table.tsx

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+// import { Map } from "immutable";
+import Markdown from "markdown-to-jsx";
+import LaunchIcon from "../../../../img/svg-icons/launch-icon.svg";
+
+import css from "../../../../css/portal-dashboard/feedback/rubric-table.less";
+interface IProps {
+  activityStarted: boolean;
+  rubric: any;
+  student: any;
+  rubricFeedback: any;
+  activityId: string;
+  activityIndex: number;
+  updateActivityFeedback?: (activityId: string, activityIndex: number, platformStudentId: string, feedback: any) => void;
+}
+export class RubricTableContainer extends React.PureComponent<IProps> {
+  render() {
+    const { activityStarted, rubric } = this.props;
+    const { criteria } = rubric;
+
+    if (activityStarted) {
+      return (
+        <div className={css.rubricContainer} data-cy="rubric-table">
+          {this.renderColumnHeaders(rubric)}
+          <div className={css.rubricTable}>
+            {criteria.map((crit: any) =>
+              <div className={css.rubricTableRow} key={crit.id} id={crit.id}>
+                <div className={css.rubricDescription}>
+                  <Markdown>{crit.description}</Markdown>
+                </div>
+                {this.renderRatings(crit)}
+              </div>
+            )}
+          </div>
+        </div>
+      );
+    }
+
+    return null;
+  }
+
+  private renderColumnHeaders = (rubric: any) => {
+    const { referenceURL } = rubric;
+    return (
+      <div className={css.columnHeaders}>
+        <div className={css.rubricDescriptionHeader}>
+          <div className={css.scoringGuideArea}>
+            <a className={css.launchButton}>
+              <LaunchIcon className={css.icon} href={referenceURL} target="_blank" />
```

Comment:
This link/button isn't working. I think the `href` needs to be moved from the icon to the `a` element.